modules/smarthome/avmhomeautomation/avmcommon.py

```diff
@@ -13,6 +14,13 @@ def getAVMSessionID(
         username, 
         previoussessionid="0000000000000000",
         ):
+
+    # fallback for storing password and credentials locally
```

Comment:
NB: this is for my local development, as it allows me to run the scripts directly via command line instead of going through openWB